packages/flutter/lib/src/widgets/image.dart

```diff
@@ -44,6 +45,15 @@ ImageConfiguration createLocalImageConfiguration(BuildContext context, { Size si
   );
 }
 
+/// Prefetches an image into the image cache.
+Future<Null> prefetchImage(ImageProvider provider, BuildContext context, { Size size }) {
+  final ImageConfiguration config = createLocalImageConfiguration(context, size: size);
```

Comment:
Image doesn't really anything to do with the caching. I mean, you could just as easily be using FadeInImage, for example. Or passing it to a BoxDecoration.